DatadogCore/Sources/Core/DatadogCore.swift

```diff
@@ -491,3 +491,12 @@ extension DatadogCore: Flushable {
         }
     }
 }
+
+#if SPM_BUILD
+import DatadogPrivate
+#endif
+
+internal let registerObjcExceptionHandlerOnce: () -> Void = {
```

Comment:
/question Do we need to make sure that `registerObjcExceptionHandlerOnce` is called only once?

Reply:
I believe this is because we support multiple SDK instances. This handler can be considered a shared resource between them. In any case, we could make it clear with minimal API comment.